src/components/molecules/Settings/SettingPage/index.tsx

```diff
@@ -157,4 +171,7 @@ const MenuIcon = styled(Icon)`
   }
 `;
 
+const StyledLoading = styled(Loading)`
+  margin: 52px auto;
+`;
```

Comment:
Please remember to space everything consistently. It makes things look better and easier to read.
```suggestion
`;

```